agent/consul/health_endpoint.proto

```diff
@@ -3,13 +3,24 @@ syntax = "proto3";
 package consul;
 
 service Health {
-    rpc Stream(TestRequest) returns (stream TestReply) {}
+    rpc Subscribe(SubscribeRequest) returns (stream UpdateEvent) {}
 } 
 
-message TestRequest {
-    string name = 1;
+message SubscribeRequest {
+    string key = 1;
 }
   
-message TestReply {
-    int32 data = 1;
+message UpdateEvent {
+    repeated ServiceHealthUpdate serviceHealthUpdate = 1;
+}
+
+message ServiceHealthUpdate {
+    string node = 1;
+    string address = 2;
+    int32 port = 3;
+    repeated HealthCheck checks = 4;
```

Comment:
Probably need a service ID as well as node to identify this uniquely. Probably also DC once we support cross-dc stuff. I'd say it's worth adding now just so we don't benchmark artificially simple solution that will only get worse as we add the full functionality.

Actually, we should probably make this be equivalent to a full `structs.CheckServiceNode` for now as that is what we'll need in order to replicate the current public health API at the client end so we should benchmark that. We can have a more optimized version for service discovery cases that don't need all the extra info (like connect proxies) later but lets build the PoC with all the fidelity we need for equivalent client API for now so we compare apples-to-apples.